Dockerfile

```diff
@@ -41,8 +41,12 @@ RUN apt-get update && \
 
 # binary downloads
 ARG TFSEC_VERSION="v0.24.1"
+ARG PACKER_VERSION="v1.6.1"
 RUN curl -L https://github.com/liamg/tfsec/releases/download/${TFSEC_VERSION}/tfsec-linux-amd64 -o /usr/local/bin/tfsec && \
-    chmod 0755 /usr/local/bin/tfsec
+    chmod 0755 /usr/local/bin/tfsec && \
+    curl -L https://releases.hashicorp.com/packer/${PACKER_VERSION#v}/packer_${PACKER_VERSION#v}_linux_amd64.zip -o /usr/local/bin/packer.zip && \
```

Comment:
I don't like injecting into URLs like this as it seems brittle, but they don't seem to have `apt` support.

Reply:
I agree, they have publicly stated they do not currently have a goal to support packaged distribution